public/controllers/settings/settings.js

```diff
@@ -703,4 +722,45 @@ export class SettingsController {
   configEquivalence(key) {
     return configEquivalences[key] || '-';
   }
+
+  /**
+   * Cancel edition of a configuration entry
+   */
+  cancelEditingKey() {
+    this.editingKey = false;
+    this.editingNewValue = '';
+  }
+
+  /**
+   * Enable edition for a given key
+   * @param {String} key Configuration key
+   */
+  setEditingKey(key, value) {
+    if (typeof value === 'object') {
+      try {
+        value = JSON.stringify(value);
```

Comment:
Why we need a string instead of an object?? We should try to avoid any stringify/parse always if it's possible